Cargo.toml

```diff
@@ -30,13 +30,11 @@ categories = ["cryptography"]
 #########################
 # ZKsync dependencies  #
 #########################
-zksync_multivm = { git = "https://github.com/matter-labs/zksync-era.git", rev = "core-v25.4.0" }
-zksync_contracts = { git = "https://github.com/matter-labs/zksync-era.git", rev = "core-v25.4.0" }
-zksync_types = { git = "https://github.com/matter-labs/zksync-era.git", rev = "core-v25.4.0" }
-zksync_vm_interface = { git = "https://github.com/matter-labs/zksync-era.git", rev = "core-v25.4.0" }
-zksync_web3_decl = { git = "https://github.com/matter-labs/zksync-era.git", rev = "core-v25.4.0", features = [
-    "server",
-] }
+zksync_multivm = "=26.2.1-non-semver-compat"
+zksync_contracts = "=26.2.1-non-semver-compat"
+zksync_types = "=26.2.1-non-semver-compat"
+zksync_vm_interface = "=26.2.1-non-semver-compat"
+zksync_web3_decl = "=26.2.1-non-semver-compat"
```

Comment:
Seems like there is no reason to keep using git revision now that we publish to crates.io so I have migrated us over